Services/DeviceClient.cs

```diff
@@ -103,13 +105,30 @@ public class DeviceClient : IDeviceClient
 
         public async Task SendMessageAsync(string message, DeviceModel.DeviceModelMessageSchema schema)
         {
-            var eventMessage = new Message(Encoding.UTF8.GetBytes(message));
+            var eventMessage = default(Message);
+
+            switch (schema.Format)
```

Comment:
add a default case, throwing an exception about unknown format

Reply:
also would refactor out to private methods (nice to have)

Reply:
+1 for refactoring into private methods